velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -287,13 +306,13 @@ TEST_F(HdfsFileSystemTest, missingFileViaReadFile) {
 TEST_F(HdfsFileSystemTest, schemeMatching) {
   try {
     auto fs = std::dynamic_pointer_cast<filesystems::HdfsFileSystem>(
-        filesystems::getFileSystem("/", nullptr));
+        filesystems::getFileSystem("s3://", nullptr));
```

Comment:
https://github.com/facebookincubator/velox/pull/9835/files#r1758102096 moved InsertIntoHdfsTest ahead HdfsFileSystemTest.  And InsertIntoHdfsTest already register the local filesystem [here](https://github.com/facebookincubator/velox/blob/main/velox/connectors/hive/storage_adapters/hdfs/tests/InsertIntoHdfsTest.cpp#L38). So i changed the test from `/` to `s3` path to catch the `No registered file system matched with file path` exception.